test/Test/Database/LSMTree/Internal/RunAcc.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE BangPatterns  #-}
 {-# LANGUAGE TupleSections #-}
 
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+
```

Comment:
That's true, but I prefer uni-patterns because it means that if it does fail in a test, you get a proper source location. With `fromJust` you get "error: fromJust". :facepalm:

Reply:
It's 

```
fromJust :: HasCallStack => Maybe a -> a
```

for some time being already/